### ISSUE TITLE
PACKAGE UPDATE AND INSTALLATION BUG ⚠️

### ISSUE THREAD

**Issue by skmirajulislam** [Vin205/Enyanjyoti#71]
FIX THIS BUG !
module is not supported, and leaks memory. Do not use it. Check out lru-cache if you want a good and tested way to coalesce async requests by a key value, which is much more comprehensive and powerful.

reward 25 point whoever fix it earlier!
GSSOC & HACTOBERFEST accepted!

![Screenshot 2024-10-06 103757](https://github.com/user-attachments/assets/6ec7e590-b8d8-4397-92ab-1bff3b8dc387)

**Comment by ArchanRD** [comment]
hey @skmirajulislam I have made an attempt to solve this issue. I can see that a dependency package `inflight` is causing the issue. Below is the dependency tree:

```
-- gh-pages@6.1.1
      -- globby@6.1.0
        -- glob@7.2.3
           -- inflight@1.0.6
```
I checked for updates, `gh-pages` is already on the latest version. I have tried to update `globby`, but `gh-pages` is forcing to use the older version of globby.

So I would suggest to use an alternative for gh-pages here. e.g Netlify, GitLab Pages, etc.
